doc/user/install-operator-sdk.md

```diff
@@ -86,8 +86,8 @@ $ chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p
 - [go][go_tool] version v1.13+.
 
 ```sh
-$ go get -d github.com/operator-framework/operator-sdk # This will download the git repository and not install it
-$ cd $GOPATH/src/github.com/operator-framework/operator-sdk
+$ git clone https://github.com/operator-framework/operator-sdk
+$ cd operator-sdk
```

Comment:
```suggestion
$ cd $GOPATH/src/github.com/operator-framework/operator-sdk
```

Reply:
hmm, building from within the `$GOPATH` is what seems to be causing problems for me though. i wonder if we need another step here about the proper way to configure the go modules?

when i look at `go help modules`, the advice there is to clone a repo with modules outside of the `$GOPATH` tree. it seemed like this would be the easiest way for someone to either build the tool for usage or to contribute. i will dig a little deeper into the go docs to see if there is an option to relax the go module awareness.

Reply:
It is solved .. closing as resoolved